### ISSUE TITLE
Textarea in footer?

### ISSUE THREAD

**Issue by janefaith** [makeusabrew/bootbox#419]
Hi! I was wondering if it is possible to put textarea at the bottom of the dialog box along with the buttons? (like the reply textarea in twitter direct message) 
Thanks a bunch. :)

**Comment by tiesont** [comment]
Not sure what you're asking for. If you want a textarea in the .modal-footer section, that's not directly possible with Bootbox. 

If, instead, you mean you want a dialog similar to the direct message dialog on Twitter, then yes, you can do that, using a custom dialog and supplying your own `message` option.

**Comment by stygiansabyss** [comment]
This is something I was looking for as well.  The ability to add text before the buttons.

Example:
In a custom dialog I have a button to continue and a button to cancel.  Before these I would like to have the text "Do you wish to proceed?".  Just plain text, then the two buttons.

**Comment by janefaith** [comment]
you mean like this?
[image: Inline images 1]

**Comment by stygiansabyss** [comment]
Your image didn't show up sadly :(

**Comment by janefaith** [comment]
​
i'm so sorry for the late response. here's the image I previously sent.

**Comment by stygiansabyss** [comment]
Still nothing showing up

**Comment by janefaith** [comment]
what? how can that be? let me try again.

**Comment by tiesont** [comment]
@stygiansabyss I assume you're wanting that text inline with the buttons? Not really possible with bootbox, unless you want to modify the source.

**Comment by stygiansabyss** [comment]
Yeah, something like the buttons array, but an option to just drop plain text in before them.  Do you have any idea how much effort would be involved in adding this?

**Comment by tiesont** [comment]
@stygiansabyss Not really, and what you're asking for isn't what the OP was asking for; this user wanted a textarea (an input box, if you will) inline with the buttons. Open a new issue and I'll look into it; doesn't seem to hard, but I'm not sure it's something that should make it into the plugin anyway. You can fork the project, and if I figure it out, pull the changes in for your customized version.

**Comment by makeusabrew** [comment]
Hi there - I'm declaring bug bankruptcy on a lot of old issues today as I can't keep on top of them all. If there's still an issue here please raise a new one with as clear and concise an overview as possible. Thanks!
